makePoplogCommander.sh

```diff
@@ -238,6 +238,10 @@ UTILITY ACTIONS
 poplog --help
     A special case that shows this usage information.
 
+poplog --version
+    A special case that displays the version information for GetPoplog and
+    the base Poplog system that it accompanies.
```

Comment:
How about
> Show version information for GetPoplog and the base Poplog system on standard output and exit successfully

Instead?
It's less ambiguous where the information will go and what the exit strategy is.

Reply:
Done.